tests/t5_cp_mount_helper.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+'''
+This file reproduces mount.s3ql, with pyfuse3.invalidate_inode patched.
+'''
+
+import sys
+import os.path
+
+if __name__ == '__main__':
+    basedir = os.path.abspath(os.path.join(os.path.dirname(sys.argv[0]), '..'))
+    sys.path = [os.path.join(basedir, 'src')] + sys.path
+
+    # Override pyfuse3.invalidate_inode : Drop kernel dentries and inodes cache
+    # just before calling pyfuse3.invalidate_inode
+    import pyfuse3
+    pyfuse3_invalidate_inode = pyfuse3.invalidate_inode
+    def patched_pyfuse3_invalidate_inode(inode):
+        # echo 2 > /proc/sys/vm/drop_caches : Drop kernel dentries and inodes cache
+        with open("/proc/sys/vm/drop_caches", "w") as drop_caches:
+            drop_caches.write("2\n")
```

Comment:
I'm seeing this fail when running within a container with all other prerequisites so there may need to be an additional check in here rather than just running as root